spec/support/account.rb

```diff
@@ -47,3 +47,11 @@ class Account < ActiveRecord::Base
     property :serialized, :type => :serialized
   end
 end
+
+module Other
+  class Account < ::Account
+    def self.delegated_property_set_attributes
```

Comment:
So will you have to do this everywhere?

Reply:
This is a copy of something we have to do in the main app. I went down the path of automatically including this behavior by default when using `delegate_to_property_set` but I don' think that we want to enforce that on every user of it.

The solution included making this a `class_attribute` which would be an inherited attribute for classes... but that seemed outside the scope of this module.

Reply:
Got it. I saw it in Classic and here and feared it would be necessary in all situations.